tests/test_destalinator.py

```diff
@@ -387,7 +387,7 @@ def test_skips_archiving_if_before_earliest_archive_date(self, mock_slacker):
         self.destalinator.archive = mock.MagicMock(return_value=True)
         mock_slacker.channel_has_only_restricted_members.return_value = False
         today = datetime.date.today()
-        self.destalinator.earliest_archive_date = today.replace(day=today.day + 1).isoformat()
+        self.destalinator.earliest_archive_date = today.replace(day=today.day + 1)
```

Comment:
I think @fatbusinessman's changes around earliest archive date changed it from a string to a datetime, so I had to drop isoformat calls.